src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewModel.swift

```diff
@@ -128,35 +128,27 @@ class HomeTableViewModel {
 
 	func updateTestResult() {
 		// According to the tech spec, test results should always be updated in the foreground, even if the final test result was received. Therefore: force = true
-		coronaTestService.updateTestResult(for: .pcr, force: true) { [weak self] result in
-			guard let self = self else { return }
-
-			if case .failure(let error) = result {
-				switch error {
-				case .noCoronaTestOfRequestedType, .noRegistrationToken, .testExpired:
-					// Errors because of no registered corona tests or expired tests are ignored
-					break
-				case .responseFailure, .serviceError, .registrationTokenError, .unknownTestResult, .malformedDateOfBirthKey:
-					// Only show errors for corona tests that are still expecting their final test result
-					if self.coronaTestService.pcrTest != nil && self.coronaTestService.pcrTest?.finalTestResultReceivedDate == nil {
-						self.testResultLoadingError = error
-					}
-				}
-			}
-		}
-
-		coronaTestService.updateTestResult(for: .antigen, force: true) { [weak self] result in
-			guard let self = self else { return }
-
-			if case .failure(let error) = result {
-				switch error {
-				case .noCoronaTestOfRequestedType, .noRegistrationToken, .testExpired:
-					// Errors because of no registered corona tests or expired tests are ignored
-					break
-				case .responseFailure, .serviceError, .registrationTokenError, .unknownTestResult, .malformedDateOfBirthKey:
-					// Only show errors for corona tests that are still expecting their final test result
-					if self.coronaTestService.antigenTest != nil && self.coronaTestService.antigenTest?.finalTestResultReceivedDate == nil {
-						self.testResultLoadingError = error
+		CoronaTestType.allCases.forEach { coronaTestType in
+			Log.info("Updating result for test of type: \(coronaTestType)")
+			coronaTestService.updateTestResult(for: coronaTestType, force: true) { [weak self] result in
+				guard let self = self else { return }
+
+				if case .failure(let error) = result {
+					switch error {
+					case .noCoronaTestOfRequestedType, .noRegistrationToken, .testExpired:
+						// Errors because of no registered corona tests or expired tests are ignored
+						break
+					case .responseFailure(let responseFailure):
+						switch responseFailure {
+						case .fakeResponse:
+							Log.info("fake response - skip it's not an error")
```

Comment:
Perhaps increase from `info` to `warn`?

Reply:
basically we only log this for info purpose - I don't think this is a warning.